set.mm

```diff
@@ -748683,6 +748793,185 @@ set of all the singletons of (proper) ordered pairs over the elements of
+        GUUAXTAUUAGABCFGHIKLMNOVGUSZVIAKFVJZUUBYPVKAFGKVLZUUDABCFGHIKLMNOVMZFGK
+        VNVTZUCFXTKVOVTVPAYLYBUUAVEZYTAGUUAYBUUCVIAUUDUUHYTVKUUGUDFYBKVOVTVPVQA
+        YPYTYFAYOYTYFTUCFAYMFVEZVFZYTYOYFUUJYSYOYFTUDFUUJYQFVEZVFZYSYOYFUULYNJU
+        IZYRJUIZSZYNYRSZTZYSYOVFZYFAUUIUUKUUQAFEIHVRZUEZUUIUUKVFZUUQTZRUUTFEUUS
+        UGZXTUUSUIZYBUUSUIZSZYETZUBFUJUAFUJZVFZAUVBUAUBFEUUSVSZAUVHUVBUVCAUVAUV
+        HUUQAUVAUVHUUQTAUVAVFZUVHYMUUSUIZYQUUSUIZSZYMYQSZTZUUQUVAUVHUVPTAUVGUVP
+        UVLUVESZYMYBSZTUAUBYMYQFFXTYMSUVFUVQYEUVRXTYMUVEUUSWAXTYMYBWBWCYBYQSZUV
+        QUVNUVRUVOUVSUVEUVMUVLYBYQUUSWDWEYBYQYMWFWCWGWJUVKUVPUUOUVOTUUQUVKUVNUU
+        OUVOUVKUVLUUMUVMUUNAUUIUVLUUMSUUKABCDEFGHIJKYMLMNOPQWHWIAUUKUVMUUNSUUIA
+        BCDEFGHIJKYQLMNOPQWHWKWLWMUVKUVOUUPUUOUVOUUPTUVKYMYQKWDUPWNWOXAWPWQWRWS
+        WTXBUURUUOYDUUPYEYOYSUUMYAUUNYCYNXTJWDYRYBJWDXCYOYSUUPYEVKYNXTYRYBXDXJW
+        CXEXFXGWQXGXKWOXHUAUBGEJVSXIAFGKUEZUUEXRAFGKUGZXTKUIZYBKUIZSZYETZUBFUJU
+        AFUJZUVTAUUEUWAUUFFGKXLVTAUUTUWFRUUTUVIAUWFUVJAUVHUWFUVCAUVGUWEUAUBFFAX
+        TFVEZYBFVEZVFVFZUVGUWBJUIZUWCJUIZSZYETUWEUWIUVFUWLYEUWIUVDUWJUVEUWKAUWG
+        UVDUWJSUWHABCDEFGHIJKXTLMNOPQWHWIAUWHUVEUWKSUWGABCDEFGHIJKYBLMNOPQWHWKW
+        LWMUWIUWDUWLYEUWDUWLTUWIUWBUWCJWDUPXMWOXNWRWSWTUAUBFGKVSXIUUFFGKXOXIXP
+        $.
+    $}
+
+    ${
+      fcoresfo.s $e |- ( ph -> ( G o. F ) : P -onto-> D ) $.
+      $( If a composition is surjective, then the minimum restriction of its
+         first component is surjective.  (Contributed by AV, 17-Sep-2024.) $)
+      fcoresfo $p |- ( ph -> X : E -onto-> D ) $=
+        ( wf wfo ccom cres crn cdm cin wss a1i wceq fdmd eqcomd 3sstr4d fssresd
+        inss2 feq1i sylibr fcoreslem3 fof syl fcores eqidd mpbird foco2 syl3anc
+        foeq123d ) AGEJSZFGKSZFEJKUAZTZGEJTAGEIGUBZSVEADEGIPAHUCZIUDZUEZVKGDVLV
+        KUFAVJVKUMUGGVLUHAMUGAVKDADEIPUIUJUKULGEJVIQUNUOAFGKTVFABCFGHIKLMNOUPFG
+        KUQURAVHFEIHUAZTRAFFEEVGVMAVMVGABCDEFGHIJKLMNOPQUSUJAFUTAEUTVDVAFGEJKVB
+        VC $.
+    $}
+
+    ${
+      f1cof1blem.s $e |- ( ph -> ran F = C ) $.
+      $( Lemma for ~ f1cof1b and ~ focofob .  (Contributed by AV,
+         18-Sep-2024.) $)
+      f1cof1blem $p |- ( ph -> ( ( P = A /\ E = C )
+                              /\ ( X = G /\ Y = F ) ) ) $=
+        ( wceq eqtrd wa ccnv crn cima fcoreslem1 fdmd eqtr4d imaeq2d cnvimarndm
+        cdm syl5eq cin simpr adantr ineq12d inidm eqtrdi mpdan jca cres reseq2d
+        wrel freld resdm syl jca32 ) AFBSZGDSZUAJISKHSAVGVHAFHUBZHUCZUDZBAFVIIU
+        JZUDVKABCFGHILMNUEAVLVJVIAVLDVJADEIPUFZRUGUHTZAVKHUJZBHUIZABCHLUFUKTAGV
+        JVLULZDMAVJDSZVQDSRAVRUAZVQDDULDVSVJDVLDAVRUMAVLDSVRVMUNUODUPUQURUKZUSA
+        JIVLUTZIAJIGUTWAQAGVLIAGDVLVTVMUGVAUKAIVBWAISADEIPVCIVDVETAKHVOUTZHAKHF
+        UTWBOAFVOHAFVKVOVNVPUQVAUKAHVBWBHSABCHLVCHVDVETVF $.
+    $}
+  $}
+
+  $( The composition ` ( G o. F ) ` is injective iff ` F ` and ` G ` are both
+     injective (if the range of ` F ` equals the domain of ` G ` ).
```

Comment:
Here and below, the condition in the parentheses is essential, so I would remove the parentheses and put it first:
> If the range of ` F ` equals the domain of ` G ` , then the composition...

Reply:
I think the "iff" statement is the most important part, and usually the range of ` F ` equals the domain of ` G `.

Reply:
No, that condition is essential and should be placed first, and not in parentheses. When I did read your sentence the first time, I really startled before I could even read the parenthesis.

Reply:
OK, I revised the comments accordingly.